.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,98 @@
+name: Benchmarks
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+  workflow_dispatch:
+    branches:
+      - '*'
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java-version: ['17']
+        julia-version: ['1.7.1']
+        python-version: ['3.9']
+        numpy-version: ['1.22']
+        gfortran-version: ['9']  # Note: unused since is built-in.
+        rust-version: ['1.42.0']  # Note: unused since controlled by `rust/rust-toolchain`
+        js-version: ['16']
+        r-version: ['4.1.2']
+        lua-version: ['2.0.5']  # Note: Not used as benchmark is broken.
+        go-version: ['1.17.4']  # Note: Not used as benchmark is broken.
+    
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
```suggestion
      - uses: actions/checkout@v2
        with:
          persist-credentials: false
```

Reply:
Will add. Out of curiosity, what will this change?